tests/event_lists/test_south_african.py

```
@@ -0,0 +1,405 @@
+    "South Africa–United Arab Emirates sports relations": "العلاقات الإماراتية الجنوب الإفريقية الرياضية",
+    "South Africa–United Kingdom relations": "العلاقات البريطانية الجنوب الإفريقية",
+    "South Africa–United Kingdom sports relations": "العلاقات البريطانية الجنوب الإفريقية الرياضية",
+    "South Africa–United States relations": "العلاقات الأمريكية الجنوب الإفريقية",
+    "South Africa–Uruguay relations": "العلاقات الأوروغويانية الجنوب الإفريقية",
+    "South Africa–Venezuela relations": "العلاقات الجنوب الإفريقية الفنزويلية",
+    "South Africa–Yugoslavia relations": "العلاقات الجنوب الإفريقية اليوغسلافية",
+    "South Africa–Zambia relations": "العلاقات الجنوب الإفريقية الزامبية",
+    "South Africa–Zimbabwe border crossings": "معابر الحدود الجنوب الإفريقية الزيمبابوية",
+    "South Africa–Zimbabwe border": "الحدود الجنوب الإفريقية الزيمبابوية",
+    "South Africa–Zimbabwe relations": "العلاقات الجنوب الإفريقية الزيمبابوية",
+    "South Africa–Zimbabwe sports relations": "العلاقات الجنوب الإفريقية الزيمبابوية الرياضية",
```

Ruff (0.14.10)

Dictionary key literal `"South Africa–Zimbabwe sports relations"` repeated

Remove repeated key literal `"South Africa–Zimbabwe sports relations"`

(F601)